---
title: Software
layout: breanna
---
<br>
For a full list of our labs software, see our <a href="https://github.com/Boyle-Lab">GitHub Account</a>. <br><br>


<div class = "row color_1">
  <center><h4>REGULOME<i><b>DB</b></i></h4></center>
  <center><p class="yellow-dot"></p></center>
<img src="/assets/images/regulomedb_logo.png" width="275px" class="imgright">
<p class = "align-left">RegulomeDB is a database that annotates SNPs with known and predicted regulatory elements in the intergenic regions of the H. sapiens genome. Known and predicted regulatory DNA elements include regions of DNAse hypersensitivity, binding sites of transcription factors, and promoter regions that have been biochemically characterized to regulation transcription.</p>
<br>
<P class = "align-left"> RegulomeDB is available at <a href="http://www.regulomedb.org/">http://RegulomeDB.org/</a>.</p>
<br>
<p class = "align-left">Reference:<br>
<b>Annotation of functional  variation in personal genomes using RegulomeDB.</b>
<br><b id="me">Boyle AP</b>, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, Karczewski KJ,  Park J, Hitz BC, Weng S, Cherry JM, Snyder M. <b><em>Genome  Research</em></b> 2012, 22(9):1790-1797. PMID: 22955989.
</p>
</div>

<br>

<div class = "row blue">
  <center><h4><font color = #FFCB05>SELF ORGANIZING <i><b>MAPS</b></i></font></h4></center>
  <center><p class="yellow-dot"></p></center>
<img src="/assets/images/SOMcluster-01.png" width="250px" class="imgright">
<p class = "align-left"><font color = white>
We use self organizing maps to explore the combinatorial space of transcription factor binding. This method allowed us to explore the full space of 128 transcription factors as well as compare patterns across different cell-types and species.
</font>
</p>
<br>
<P class = "align-left"><font color = white> Modified R package and example code is available at <a class = "two" href="http://gbsc-share.stanford.edu/aboyle/index.html">http://gbsc-share.stanford.edu/aboyle/index.html</a>.<br>
Mouse SOM browser: <a class = "two" href="https://boylelab.med.umich.edu/SOMbrowser">https://boylelab.med.umich.edu/SOMbrowser</a><br>
Comparative SOM analysis tools: <a class = "two" href="https://github.com/Boyle-Lab/SOM-Browser">https://github.com/Boyle-Lab/SOM-Browser</a>
</font>
</p>
<br>
<p class = "align-left"><font color = white>References:<br>
<b>Dynamic trans-acting  factor colocalization in human cells.</b>
<br>*Xie D, <b id="me">*Boyle AP</b>, *Wu L, Kawli T, Zhai J, Snyder M. <b><em>Cell</em></b> 2013,  155(3):713-724. PMID: 24243024.
<br>Diehl AG, <b id="me">Boyle AP</b>. <b><em>Nucleic Acids Research</em></b> 2018, :gky018. PMID: 
</font>
</p>
</div>

<br>

<div class = "row color_1">
  <center><h4>F-<i><b>SEQ</b></i></h4></center>
  <center><p class="yellow-dot"></p></center>
<img src="/assets/images/Fseq.gif" width="250px" class="imgright">
<p class = "align-left">
To intuitively summarize and display individual sequence data as an accurate and interpretable signal, we developed F-Seq, a software package that generates a continuous tag sequence density estimation allowing identification of biologically meaningful sites whose output can be displayed directly in the UCSC Genome Browser.
  </p>
<br>
<P class = "align-left">F-seq is available at <a href="https://github.com/aboyle/F-seq">https://github.com/aboyle/F-seq</a> and detailed documentation is available at <a href="http://fureylab.web.unc.edu/software/fseq/">http://fureylab.web.unc.edu/software/fseq/</a>.</p>
<br>
<p class = "align-left">Reference:<br>
<b>F-Seq: a feature density  estimator for high-throughput sequence tags.</b>
<br><b id="me">Boyle AP</b>, Guinney J, Crawford GE, Furey TS. <b><em>Bioinformatics</em></b>  2008, 24(21):2537-2538. PMID: 18784119.
</p>
</div>
